system_hooks/src/addresses_constants.rs

```diff
@@ -14,6 +14,8 @@ pub const ECPAIRING_HOOK_ADDRESS_LOW: u16 = 0x0008;
 pub const BLAKE_HOOK_ADDRESS_LOW: u16 = 0x0009;
 #[cfg(feature = "mock-unsupported-precompiles")]
 pub const POINT_EVAL_HOOK_ADDRESS_LOW: u16 = 0x000a;
+#[cfg(feature = "p256_precompile")]
+pub const P256_VERIFY_PREHASH_HOOK_ADDRESS_LOW: u16 = 0x000b;
```

Comment:
Let's use some other address, maybe a bit further away. We don't support it, but 0xb -0x11 are taken here https://eips.ethereum.org/EIPS/eip-2537